pkg/environment/api/organization.go

```diff
@@ -396,6 +635,95 @@ func (s *EnvironmentService) createOrganization(
 	return nil
 }
 
+// Create a default project.
+// We must create a project when creating an Organization
+// because we also need to create the organization owner in the account table.
+// To create it we need the project, so we can also create the environment.
+func (s *EnvironmentService) createDefaultProject(
+	ctx context.Context,
+	tx mysql.Transaction,
+	organizationID, email string,
+) (*domain.Project, error) {
+	project, err := domain.NewProject(
+		"Default Project",
+		"default",
+		"",
+		email,
+		organizationID,
+		true,
+	)
+	if err != nil {
+		return nil, err
+	}
+	projectStorage := v2es.NewProjectStorage(tx)
+	if err := projectStorage.CreateProject(ctx, project); err != nil {
+		return nil, err
+	}
+	return project, nil
+}
+
+// Create Development and Production default environments.
+// We must create the environment when creating an Organization
+// because we also need to create the organization owner in the account table
+// and to create it we need the organization and environment roles.
+func (s *EnvironmentService) createDefaultEnvironments(
+	ctx context.Context,
+	tx mysql.Transaction,
+	organizationID string,
+	project *domain.Project,
+) ([]*accountproto.AccountV2_EnvironmentRole, error) {
+	envRoles := make([]*accountproto.AccountV2_EnvironmentRole, 0, 2)
+	envNames := []string{
+		"Development",
+		"Production",
+	}
+	for _, name := range envNames {
+		envURLCode := fmt.Sprintf("%s-%s", project.UrlCode, strings.ToLower(name))
```

Comment:
The environment must have a unique URL code. Otherwise, it will conflict with other environments in the same project. So, I used the project URL code as a prefix.